src/schoology-extractor/schoology_extractor/api/request_client.py

```diff
@@ -144,19 +148,31 @@ def get_section_by_id(self, section_id: str) -> dict:
             A parsed response from the server
 
         """
-        assert isinstance(section_id, str), "Argument `section_id` should be of type `str`."
+        assert isinstance(course_ids, list), "Argument `course_ids` should be of type `list`."
+
+        sections = []
+        for course_id in course_ids:
+            url = f"courses/{course_id}/sections"
+            sections_per_course = PaginatedResult(
+                self, 20, self.get(url), "section", self.base_url + url
```

Comment:
Is this hard-coding a page size of 20?

Reply:
Yes, in some situations where we have complex requests... Such as needing to merge the results from different paginated results, I opted for just returning the final list and not the paginated result.

Reply:
I admit I don't understand that. Just looking at the code, it looks like a bug. This might be a good time to replace `20` with a well-named constant (https://refactoring.guru/replace-magic-number-with-symbolic-constant), and possibly add a text comment explaining why this is being done.

Reply:
I will

Reply:
✅